wrappers/wtsapi32.go

```diff
@@ -201,6 +202,24 @@ func WTSQuerySessionInformation(handle syscall.Handle, sessionId uint32, infoCla
 	return nil
 }
 
+func WTSLogoffSession(handle syscall.Handle, sessionId uint32, wait bool) (bool, error) {

```

Comment:
No need for a Boolean in the return value if it just distinguishes between success and failure.  The caller can tell the difference by just checking whether the error is nil or not.